app/api/business/brief/brief_edit_business.py

```diff
@@ -214,6 +229,16 @@ def edit_opportunity(user_id, brief_id, edits):
 
         errors.append(message)
 
+    if (
+        closing_date_was_edited(brief.closed_at.to_iso8601_string(), previous_data['closed_at']) and
+        not validator.validate_closed_at_blackout()
+    ):
+        message = (
+            'The closing date must not be during the transition period to BuyICT'
```

Comment:
Do we need to specify the start and end of the blackout period in this message?

Reply:
Yes, I think it would be a good idea. I have included the start & end dates in a newer commit.